packages/clients/src/wormholeClient.ts

```diff
@@ -0,0 +1,98 @@
+import { BigNumber } from 'ethers';
+import { ChainID, WormholeOperationsDetailsReturn } from './types';
+import { NetworkId } from '@colony/colony-js';
+import { output } from '@joincolony/utils';
+
+export class WormholeClient {
+  private readonly bridgeEndpoint: string;
+
+  public REQ_STATUS = {
+    CONFIRMED: 'confirmed',
+  };
+
+  constructor(bridgeEndpoint: string) {
+    this.bridgeEndpoint = bridgeEndpoint;
+  }
+
+  /**
+   * Pads a hex string to a specific byte length.
+   * @param hexString - The hexadecimal string to pad.
+   * @param byteLength - The target byte length (default is 32).
+   * @returns The padded hexadecimal string.
+   */
```

Comment:
These method comments are pretty nice tbh, and it might be nice at some point to go back and add these to the previous classes which you guys made!